feat(nft-commitment-query-by-denom): Enhanced getaccountcommitments  to support querying coin commitments with an optional asset filter. (#2334)
provenance-io/provenance: x/exchange/spec/05_queries.md

## GetAccountCommitments

To look up the amounts an account has committed to any market, use the `GetAccountCommitments` query.

> You can optionally filter the results by a specific asset denomination using the `asset` query parameter.

Reviewer: ```suggestion
You can optionally filter the results for a specific denomination using the `denom` query parameter.
```
